src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.ActivityDaoDb;
+import core.basesyntax.dao.ActivityDaoDbImpl;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.storage.Storage;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportServiceImplTest {
+
+    private static final String CSV_SEPARATOR = ",";
+    private static final String CSV_HEAD_REPORT = "fruit,quantity";
+    private static final String CSV_END_LINE = System.lineSeparator();
+    private static ActivityDaoDb activityDaoDb;
+    private static Fruit fruitFirst;
+    private static Fruit fruitSecond;
+    private static ReportService reportService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        activityDaoDb = new ActivityDaoDbImpl();
+        reportService = new ReportServiceImpl(activityDaoDb);
+        fruitFirst = new Fruit("peach");
+        fruitSecond = new Fruit("grape");
+    }
+
+    @Test
+    public void test_empty_report() {
```

Comment:
```suggestion
    public void makeReport_emptyStorage_ok() {
```
better follow this pattern
`methodNameThatYouAreTesting_stateThatYouProvide_okOrNotOk`

Reply:
ok